File: Web.Admin/ClientApp/src/app/models/operatingEnvironment/OperatingEnvironment.ts
import { DependencyLite } from '../dependency/Dependency-lite';

export class OperatingEnvironment {

  id: number;
  name: string;
  dependencies: DependencyLite[];

  public OperatingEnvironment(id: number,
    name: string,
    dependencies: DependencyLite[]) {
    this.id = id;
    this.name = name;
    this.dependencies = dependencies;
  };

}

File: Libraries/Shared/NIST.CVP.Libraries.Shared.ACVPCore.Abstractions/Models/OperatingEnvironment.cs
using System.Collections.Generic;

namespace NIST.CVP.Libraries.Internal.ACVPCore.Models
{
    public class OperatingEnvironment
    {
        public long ID { get; set; }
        public string Name { get; set; }
        public List<DependencyLite> Dependencies { get; set; } = new List<DependencyLite>();
    }
}
